atomate/vasp/firetasks/run_calc.py

```diff
@@ -147,6 +148,11 @@ def run_task(self, fw_spec):
         elif job_type == "full_opt_run":
             jobs = VaspJob.full_opt_run(vasp_cmd, auto_npar=auto_npar, ediffg=self.get("ediffg"),
                                         max_steps=5, half_kpts_first_relax=False)
+        elif job_type == "neb":
```

Comment:
@shyuep you are correct, half_kpts was never defaulted to True, it must have been a different parameter I was thinking of.

As for the "out the window", there is a bit of subtlety here:
- for things like the automatic preset workflows, the spirit is more like what you said - i.e., just do whatever recipe you think is best. If that means half_kpts, so be it - these are details the user doesn't necessarily want to worry about.
- however, the original intention of RunVaspCustodian (the Firework object) was to be used as a drop-in replacement for RunVaspDirect. If one used the latter, VASP would be directly called. If you did a drop-in replacement of RunVaspDirect->RunVaspCustodian, the user expectation (from my point of view) is that everything should behave essentially the same as running VASP directly except you would get error correction for free. However this only really pertains to job_type="normal" - my interpretation of that is "VASP with error correction". For job types other than "normal" (e.g. NEB), this drop-in replacement thing is indeed out the window and one is free to do whatever since those not-normal job types are like mini workflows.